openapi.yaml

```diff
@@ -9812,25 +9810,22 @@ paths:
         content:
           application/json:
             schema:
-              type: object
+              allOf:
+              - $ref: '#/components/schemas/Firewall'
               required:
-                - label
-                - rules
+              - label
+              - rules
+              - inbound_policy
+              - outbound_policy
+              - action
```

Comment:
Do these need to be indented slightly more?

Reply:
We don't need to, though it looks like indents in these cases do follow correct yaml conventions. We've been historically inconsistent with this. Would rather fix them all in one go if I'm going to do it.